docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,195 @@
+  values in Parquet files. ({issue}`25667`)
+* Fix `SHOW SCHEMAS` failure when catalog names contain mixed case letters. ({issue}`25701`)
+
+## Druid connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## DuckDB connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Elasticsearch connector
+
+## Exasol connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Faker connector
+
+## Google Sheets connector
+
+* Add support for authentication using delegated user credentials with the 
+  `gsheets.delegated-user-email` configuration property. ({issue}`25746`)
+
+## Hive connector
+
+* Add support for excluding certain tables from the directory listing cache 
+  with the `hive.file-status-cache.excluded-tables` configuration property. ({issue}`25715`)
+* Add signer type support to the native S3 filesystem. ({issue}`25820`)
+* Improve compatibility with S3-compliant storage systems. ({issue}`25791`)
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Improve the directory listing cache invalidation behavior to avoid invalidating the entire
+  cache when dropping a partitioned table. ({issue}`25749`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
+* Reduce S3 throttling failures by increasing streaming upload part size and reducing number
+  of requests. ({issue}`25781`)
+* Fix query failures when attempting to read `date` columns stored as integer
+  values in Parquet files. ({issue}`25667`)
+
+## Hudi connector
+
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Fix query failures when attempting to read `date` columns stored as integer
+  values in Parquet files. ({issue}`25667`)
+
+## Iceberg connector
+
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Add signer type support to the native S3 filesystem. ({issue}`25820`)
+* Add `added_delete_files_count`, `existing_delete_files_count`, `deleted_delete_files_count`, and
+  `reference_snapshot_id` columns to `$all_manifests` metadata tables. ({issue}`25867`)
+* Improve compatibility with S3-compliant storage systems. ({issue}`25791`)
+* Show detailed metrics from splits generation in output of `EXPLAIN ANALYZE VERBOSE`. ({issue}`25770`)
+* Add `max_partitions_per_writer` session property, which corresponds to the 
+  `iceberg.max-partitions-per-writer` configuration property. ({issue}`25662`)
+* Improve query planning performance when reading from materialized views. ({issue}`24734`)
+* Prevent rare failure when `iceberg.bucket-execution` is enabled. ({issue}`25125`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
+* Reduce S3 throttling failures by increasing streaming upload part size and reducing 
+  number of requests. ({issue}`25781`)
+* Fix query timeout errors due to concurrent writes on tables with large number of 
```

Comment:
@raunaqmorarka what `writes` mean hear. Looks like the PR is about some lock contention, right?

Reply:
Writes in iceberg update some metadata at the end in the coordinator, this operation was earlier using the same threadpool as the one used by query planning and blocking those if the write was large enough. Now we separated the threadpool for query planning.